src/definitions.ts

```diff
@@ -47,6 +47,7 @@ export enum ServiceAddresses {
   STORAGE_AVAILABLE_CAPACITY = '/storage/v0/availableCapacity',
   TRIGGERS_STAKES = '/triggers/v0/stakes',
   TRIGGERS_PROVIDERS = '/triggers/v0/providers',
+  TRIGGERS_PLANS = '/triggers/v0/offers',
```

Comment:
This little bit confusing. Can it be TRIGGERS_OFFERS?

Reply:
ah of course... not sure what happened there :s